docs/modules/ROOT/pages/extensions/microprofile-health.adoc

```diff
@@ -0,0 +1,58 @@
+[[microprofile-health]]
+= MicroProfile Health Extension
+
+*Available as of Camel Quarkus version 0.21*
+
+The MicroProfile Health extension bridges the Camel health check API with MicroProfile Metrics & SmallRye Health.
+
+Maven users will need to add the following dependency to their `pom.xml` for this extension.
+
+[source,xml]
+------------------------------------------------------------
+<dependency>
+    <groupId>org.apache.camel.quarkus</groupId>
+    <artifactId>camel-quarkus-microprofile-health</artifactId>
+    <version>${camel-quarkus.version}</version>
```

Comment:
Aren't we (or shouldn't we if we don't) generally encouraging the users to import our BOM? So maybe we could omit the version here.

Reply:
Yeah, I did ponder whether to bother including the version. I'm happy to remove it.